scripts/cluster-scanner/internal/search_old_clusters.go

```diff
@@ -0,0 +1,30 @@
+package internal
+
+import (
+	"fmt"
+	"log/slog"
+	"time"
+
+	"github.com/spectrocloud/palette-sdk-go/api/models"
+)
+
+func SearchOldClusters (clusters []*models.V1SpectroClusterSummary, logger *slog.Logger) bool {
```

Comment:
```suggestion
func SearchOldClusters (clusters []*models.V1SpectroClusterSummary, logger *slog.Logger) (bool, string) {
```

Reply:
See my comment further down about returning the message instead of printing it in this function.